lib/chef.rb

```diff
@@ -32,3 +32,5 @@
 require "chef/handler/json_file"
 require "chef/event_dispatch/dsl"
 require "chef/chef_class"
+
+require "chef/dist"
```

Comment:
so this should actually go in the individual files.  not everything does a `require "chef"` at the top of it and we support stuff like `require "chef/node"` or other randomish things that have never quite been put into any kind of coherent public API.  so the general rule is that if there are class constants in the file, then there should be a require line at the top of the file.

Reply:
done!